cfme/cloud/instance/__init__.py

```diff
@@ -306,54 +303,133 @@ def power_control_from_cfme(self, *args, **kwargs):
         view.toolbar.power.item_select(kwargs.get('option'),
                                        handle_alert=not kwargs.get('cancel', False))
 
+    @property
+    def vm_default_args(self):
+        """
+        Represents dictionary used for Vm/Instance provision with minimum required default args
+        """
```

Comment:
Use docstrings formatting consistently. We use google-style docstrings in CFME. See https://gist.github.com/mfalesni/11694e0d563c80c3917ce6012400b93f
In short:
```python
"""Summary line

More details.
"""
```

This applies to other docstrings as well.